apis/identity/v1alpha1/referencers.go

```diff
@@ -1,9 +1,12 @@
 /*
-Copyright 2019 The Crossplane Authors.
+Copyright 2020 The Crossplane Authors.
+
```

Comment:
```suggestion
Copyright 2019 The Crossplane Authors.
```

We don't make changes on licenses of existing files.